pipeline/metadata/5.3.yaml

```diff
@@ -1152,3 +1152,19 @@
     - cephfs
     - upgrades
     - stage-1
+
+- name: "test-rgw-singlesite-with-archive"
+  suite: "suites/pacific/rgw/tier-2_rgw_archive.yaml"
+  global-conf: "conf/pacific/rgw/rgw_archive.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - schedule
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
```

Comment:
for schedule run looks like there are already 7 suites under tier-2 stage-2,
Can we please choose some other stage?

Reply:
@ckulal added to tier-2 stage-3